api/hypershift/v1beta1/nodepool_types.go

```diff
@@ -157,7 +157,7 @@ type NodePoolSpec struct {
 	// https://github.com/openshift/cluster-node-tuning-operator/blob/2c76314fb3cc8f12aef4a0dcd67ddc3677d5b54f/pkg/apis/tuned/v1/tuned_types.go
 	//
 	// Each ConfigMap must have a single key named "tuned" whose value is the
```

Comment:
can you please also add the link to the performanceProfile API? the one above points only to tuned